src/pipeline/node/DetectionParser.cpp

```
@@ -70,6 +72,90 @@ const NNArchiveVersionedConfig& DetectionParser::getNNArchiveVersionedConfig() c
     return archiveConfig.value();
 }
 
+void resetParser(DetectionParserOptions& parser) {
+    // Reset per-head YOLO overrides before applying optional metadata.
```

Cppcheck (2.20.0)

[error] There is an unknown macro here somewhere. Configuration is required. If DEPTHAI_NLOHMANN_DEFINE_TYPE_INTRUSIVE is a macro then please configure it.

(unknownMacro)